nats-on-kubernetes/minimal-setup.md

```diff
@@ -0,0 +1,69 @@
+# Basic NATS and NATS Streaming setup in K8S
+
+## Minimal NATS and NATS Streaming setup
+
+To try NATS with the minimal components, you can start with the following:
+
+```sh
+# Single server NATS
+kubectl apply -f https://raw.githubusercontent.com/nats-io/k8s/master/nats-server/single-server-nats.yml
+
+kubectl apply -f https://raw.githubusercontent.com/nats-io/k8s/master/nats-streaming-server/single-server-stan.yml
+```
+
+This will setup for you:
```

Comment:
I'm wondering if it might read clearer to have one section for core NATS, and another section for those that wish to use NATS streaming (after installing core NATS).

Reply:
Was thinking about that but need to explain again about NATS when installing NATS Streaming, want to try keeping them together first...